samigo/samigo-app/src/webapp/jsf/template/templateIndex.jsp

```diff
@@ -39,12 +39,12 @@
   <h:form id="templateCreateForm">
   <!-- HEADINGS -->
   <%@ include file="/jsf/template/templateHeadings.jsp" %>
-  <h:messages styleClass="messageSamigo" rendered="#{! empty facesContext.maximumSeverity}" layout="table"/>
-<div class="page-header">
-    <h1>
-       <h:outputText value="#{templateMessages.index_templates}"/>
-    </h1>
-</div>
+  <h:messages styleClass="sak-banner-error" rendered="#{! empty facesContext.maximumSeverity}" layout="table"/>
```

Comment:
The page heading has been reverted and the conflict message is still showing.

Reply:
I'll try to change that right now

Reply:
I'm not sure if it's fixed? I rebased and re-pushed, but there wasn't anything about conflicts when I was pushing it.

Reply:
Line 47 contains `>>>>>>> SAK-42183 Samigo: Standardized banners and replaces the messageSamigo ones`, which is from the rebase having a conflict. Lines 44 to 46 (in green on GitHub) are replacing lines 43 to 47 (in red on GitHub), which is reverting the work done in the header PRs. 

Here, you need to remove the `h3` tags, the conflict message (`>>>>>>> SAK-42183 Samigo: Standardized banners and replaces the messageSamigo ones`), and restore the `.page-header` div.

Reply:
Is this better? Sorry for all the confusion.

Reply:
@fostersdesign :)

Reply:
Yes, thank you. This looks better.